addons/modules/functions/fnc_moduleTracers.sqf

```diff
@@ -0,0 +1,128 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Zeus module function create a tracers effect.
+ *
+ * Arguments:
+ * 0: Logic <LOGIC>
+ * 1: Weapon <STRING>
+ * 2: Magazine <STRING>
+ * 3: Burst Delay <ARRAY>
+ * 4: Dispersion <NUMBER>
+ * 5: Target <OBJECT|ARRAY>
+ * 6: Vector To Target <ARRAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [LOGIC, _weapon, _magazine, _delay, _dispersion, _target, _vectorToTarget] call zen_modules_fnc_moduleTracers
+ *
+ * Public: No
+ */
+
+params ["_logic", "_weapon", "_magazine", "_delay", "_dispersion", "_target", "_vectorToTarget"];
+
+// Broadcast tracer parameters so they available if the module is edited
+_logic setVariable [QGVAR(tracersParams), [_weapon, _magazine, _delay, _dispersion, _target], true];
+
+// Create random dispersion array from dispersion index
+_dispersion = [0.001, 0.01, 0.05, 0.15, 0.3] select _dispersion;
+_dispersion = _dispersion;
```

Comment:
The old `[-_dispersion, 0, _dispersion]` was correct?

Reply:
Yeah it was. Sorry! Reverted 
ea6b1dd